### DISCUSSION TITLE
2.8执行无反应

### DISCUSSION THREAD

**Discussion by Sleep312** [pjy612/SteamManifestCache#13] [Question / Answer]
如图 运行库已装 2.7版本能正常执行但显示版本已禁用
(https://github.com/pjy612/SteamManifestCache/assets/149823327/cf54bf9c-801e-45c7-8bc6-b0b19ff33c32)

**Comment by pjy612** [comment]
是 13个小时内 下载的吗？中午之后？

**Reply by Sleep312** [reply]
对的

**Reply by Sleep312** [reply]
晚上刚下呢

**Reply by pjy612** [reply]
明天 再改一下吧

**Reply by Sleep312** [reply]
好的 麻烦大佬了

**Reply by pjy612** [reply]
已传 2.8.1

**Reply by Sleep312** [reply]
感谢老哥 已成功执行
